code/modules/client/preference_setup/loadout/loadout_general.dm

```diff
@@ -2,6 +2,14 @@
 	display_name = "cane"
 	path = /obj/item/cane
 
+/datum/gear/cane
+	display_name = "crutch"
+	path = /obj/item/cane/crutch
+
+/datum/gear/cane
```

Comment:
`/datum/gear/cane` is already defined, you need to make these subtypes.

Reply:
it would be best to make this a selection

Reply:
oops missed that